system_strings/sysmes_text.en

```diff
@@ -4917,167 +4982,269 @@ I feel her blatant animosity^and bloodlust. I recognize^her as an enemy. I'll lu
+ You know what they say about curiosity and cats.
 }
 [sha:1a737210c53ad44e997ac45745dc9c03d6584000]{
 -- ついに約束の時間がやって来た。^喜び勇んで訪れた茶道室には^謎の物体が……。
-ついに約束の時間がやって来た。^喜び勇んで訪れた茶道室には^謎の物体が……。
+// @flowchart_descr
+The promised time has finally arrived.
+ I'm so excited to visit the clubroom that I could hardly contain myself.
+ When I enter, I catch sight of a mysterious object...
 }
 [sha:a0a515b1c8a381097f164162ba13999e6767aa64]{
 -- 先輩の話は続く。^アルクェイドに関わる危険性と、^ロアという名の吸血鬼の事――
-先輩の話は続く。^アルクェイドに関わる危険性と、^ロアという名の吸血鬼の事――
+// @flowchart_descr
+Senpai's lecture continues.^
+Just how dangerous is Arcueid, and who is the vampire named Roa?
 }
 [sha:ec49a17e431c28ab5edd41c0803ba97865b20b22]{
 -- 先輩とは絶交をした。^気分が晴れないままの放課後、^意外な相手からメールが届く。
-先輩とは絶交をした。^気分が晴れないままの放課後、^意外な相手からメールが届く。
+// @flowchart_descr
+Ciel-senpai and I have had a falling-out.
+ As I gloomily start to head home, I receive a text from someone unexpected.
 }
 [sha:4f5f970d29ace4e3b43d98b17f8a3c4c71887701]{
 -- 気分転換になるだろう。^指定された廃工場へと足を運び、^促されるまま扉を開けた。
-気分転換になるだろう。^指定された廃工場へと足を運び、^促されるまま扉を開けた。
+// First part from 22850.
+// @flowchart_descr
+This could be a nice change of pace.
+ I make my way to the abandoned factory, and open the door as instructed.
 }
 [sha:33b5d8137384f5e8f133e264644f945488093b9b]{
 -- 因果応報、逃れる道はない。^吸血鬼を殺すと決めた以上、^この手は血に濡れていく。
-因果応報、逃れる道はない。^吸血鬼を殺すと決めた以上、^この手は血に濡れていく。
+// 因果応報 matched to 42720.
+// @flowchart_descr
+I will give him his just desserts.
+ There is no going back now.
+ The moment I resolved to kill a vampire, my hands were stained with blood.
 }
 [sha:6d3521330894be81fbce4e6c45e1ea3400b04eda]{
 -- 悪趣味には付き合いきれない。^だが、その拒絶は彼女にとって^納得のいくものではなかった。
-悪趣味には付き合いきれない。^だが、その拒絶は彼女にとって^納得のいくものではなかった。
+// Cribbed from 23021.
+// @flowchart_descr
+This is messed up. I want to part in it.
+ But Noel isn't satisfied at all by my refusal.
 }
 [sha:74a545322db885f4f945b43d9309406b93045bb5]{
 -- ノエルとビルを出て、路地裏へ。^彼女は自分の反応を愉しむように^シエルの過去を語り始める……。
-ノエルとビルを出て、路地裏へ。^彼女は自分の反応を愉しむように^シエルの過去を語り始める……。
+// @flowchart_descr
+Noel and I exit the building into a back alley.
+ She starts to tell me about Ciel's past, delighting in my reactions.
 }
 [sha:b33017f4101f45e5763f01a2eefd88258cd57822]{
 -- 沈んだ気持ちで部屋に戻る。^先輩の忠告と、あいつとの約束。^二つの間で心が揺れている――
-沈んだ気持ちで部屋に戻る。^先輩の忠告と、あいつとの約束。^二つの間で心が揺れている――
+// @flowchart_descr
+I return to my room in low spirits.
+ Senpai issued me a warning, but I made a promise with Arcueid.
+ I can't make up my mind between the two...
 }
 [sha:853098cc87b3732c8375f362476529c8bd00d367]{
 -- 約束の時間だ。^アルクェイドにもう一度会って、^自分の気持ちを確かめよう。
-約束の時間だ。^アルクェイドにもう一度会って、^自分の気持ちを確かめよう。
+// @flowchart_descr
+The promised time has arrived.
+ I'll meet up with Arcueid one more time, and confirm how I really feel.
 }
 [sha:0254e07a578acf4d16cf7b673be4c3c9db860126]{
 -- このままではまずい。^ここははっきりと言い返して、^シエル先輩にお帰り願おう。
-このままではまずい。^ここははっきりと言い返して、^シエル先輩にお帰り願おう。
+// @flowchart_descr
+I can't let things carry on like this.
+ I need to be blunt and ask Ciel-senpai to go back home.
```

Comment:
the latter sentence is partially based on Offset 23483 so add that as a comment (don't need to change anything other than that)